netsim/devices/frr.yml

```diff
@@ -152,3 +152,8 @@ features:
     vtep6: true
 
 graphite.icon: router
+
+attributes:
+  node:
+    frr:
+      debug: { type: list, split_lines: True }
```

Comment:
Could consider to make this a global attribute as well, copied under node

That way, users could define debug flags once and they would apply to all IOS nodes in the lab

Reply:
Nice one. Thank you, will add.